antelope_contracts/tests/erc20/transfer_tests.cpp

```diff
@@ -244,7 +262,7 @@ try {
 
     produce_block();
 
-    BOOST_REQUIRE_EXCEPTION(push_action(evmtok_account, "setegressfee"_n, evmtok_account, 
+    BOOST_REQUIRE_EXCEPTION(push_action(erc20_account, "setegressfee"_n, erc20_account, 
```

Comment:
I don't think we need to keep this part of the test where the authority is changed and we see that the `setegressfee` fails anymore.